packages/flutter/lib/src/widgets/routes.dart

```diff
@@ -1428,8 +1428,7 @@ abstract class ModalRoute<T> extends TransitionRoute<T> with LocalHistoryRoute<T
     }
     _ignorePointerNotifier.value = !isCurrent ||
         (navigator?.userGestureInProgress ?? false) ||
-        (ignorePointerDuringTransitions &&
-            (isTransitioning(animation) || isTransitioning(secondaryAnimation)));
+        ((isTransitioning(animation) || isTransitioning(secondaryAnimation)) && ignorePointerDuringTransitions);
```

Comment:
Reordering this so that `ignorePointerDuringTransitions` can be last, to avoid issues calling `navigator!.context` after the route has been explicitly removed from the navigator (e.g, for dropdown routes). In those cases, the `isTransitioning` methods should always prevent `ignorePointerDuringTransitions` from being evaluated